### DISCUSSION TITLE
jsonresume generated CV

### DISCUSSION THREAD

**Discussion by arunoruto** [nunocoracao/blowfish#1122] [Ideas]
[jsonresume](https://jsonresume.org/) is trying to make a standardized schema for resumes. One could fetch the resume.json file from their GitHub gist and generate a CV page. I will tackle the problem in the coming days, but I am not experienced in designing. So, while I could provide the raw data, the styling should be handled by someone more experienced 🙈

**Comment by arunoruto** [comment]
I got inspired by Nuno's resume page and came up with similar tabular data in a shortcode. An example for the [jsonresume shortcode](https://github.com/arunoruto/blowfish/blob/main/layouts/shortcodes/jsonresume.html) can be found on my webpage [here](https://www.arnaut.me/resume/).

While at it, I also created a shortcode for [orcid](https://orcid.org/), where researchers can list their works. I am still working on it, but I also have an example for the [orcid shortcode](https://github.com/arunoruto/blowfish/blob/main/layouts/shortcodes/orcid.html) on my page [here](https://www.arnaut.me/publications/).

I appreciate any feedback!

**Comment by tristam15** [comment]
can you help me create a resume like the one you have on your website? I wrote everything in markdown but my images aren't working properly. 

Help appreciated.

**Reply by arunoruto** [reply]
Sure :D

The pictures live in a folder called [resume under content](https://github.com/arunoruto/website/tree/main/content/resume). The index.md file calls my self written [short code for jsonresume](https://github.com/arunoruto/website/blob/main/layouts/shortcodes/jsonresume.html)

I used some styling right in the page, so you can basically copy and paste my template and populate it yourself without using jsonresume. I guess, I named the images accordingly to the institution and used that, to automatically attach the image :)

**Reply by tristam15** [reply]
So you created the whole thing yourself, I understood that now. 

Did everyone have to do this? How did they do this? 

I am not able to understand the concept of JSON Resume. What is it?

**Reply by tristam15** [reply]
@arunoruto I tried for several hours my friend, I couldn't get it to work. 

I finally gave up on the images and kept only text. 

Thanks for the support.

**Reply by arunoruto** [reply]
> So you created the whole thing yourself, I understood that now. 
> 
> Did everyone have to do this? How did they do this? 
> 
> I am not able to understand the concept of JSON Resume. What is it? 

[JsonResume](https://jsonresume.org/) is just a schema for resumes. You can populate the schema with your data, and different apps/frameworks can use it to create a nice resume on paper, on a website, etc. It isn't a standard, just something a few people came up with it, but I like when things have some rules we can follow :)

On their page you have a template, so you can just fill it out with your information and host it wherever you want. The folks from JsonResume recommended GitHub Gist for this!

**Reply by arunoruto** [reply]
> @arunoruto I tried for several hours my friend, I couldn't get it to work. 
> 
> I finally gave up on the images and kept only text. 
> 
> Thanks for the support. 

If you have it on your GitHub you can send me a link to the file and I can try to help you :)
